playbooks/common/openshift-master/config.yml

```diff
@@ -204,15 +204,6 @@
   - role: os_firewall
   - role: openshift_master_facts
   - role: openshift_hosted_facts
-  - role: openshift_master_certificates
-  - role: openshift_etcd_facts
-  - role: openshift_etcd_client_certificates
-    etcd_cert_subdir: "openshift-master-{{ openshift.common.hostname }}"
-    etcd_cert_config_dir: "{{ openshift.common.config_base }}/master"
-    etcd_cert_prefix: "master.etcd-"
-    r_etcd_common_etcd_runtime: "{{ openshift.common.etcd_runtime }}"
-    etcd_ca_host: "{{ groups.oo_etcd_to_config.0 }}"
-    when: groups.oo_etcd_to_config | default([]) | length != 0
```

Comment:
+1 to moving the deps out of the `openshift-master/config.yml`.

With the certificates roles removed, does it mean that when i run `playbooks/byo/openshift-master/config.yml` the certificates will not get deployed? Or is the certs deployment just missing in the byo equivalent?

Reply:
Right, the certificate playbooks are currently not included in the `config.yml` playbooks and are included one level up.